requirements/base.txt

```diff
@@ -4,19 +4,19 @@
 #
 #    make upgrade
 #
-certifi==2020.4.5.1       # via requests
+certifi==2020.11.8        # via requests
 chardet==3.0.4            # via requests
 defusedxml==0.6.0         # via python3-openid, social-auth-core
-django==2.2.12            # via -c requirements/constraints.txt, -r requirements/base.in
-idna==2.9                 # via requests
+django==2.2.17            # via -c requirements/constraints.txt, -r requirements/base.in
```

Comment:
Hmm, did we forget to add a `make upgrade` job for this, or has it been failing?

Reply:
it's scheduled to run once a week and there's a PR:https://github.com/edx/auth-backends/pull/80 created on October 2nd with no one tagged that has caused following attempts for the 'upgrade' job not to create any PRs.

Reply:
I just gave arbi-bom explicit write access to the repo so future PRs should correctly tag you for review.